client/src/bundles/UserForms.js

```diff
@@ -1,738 +1,742 @@
+      $input.closest('.field[id]').each(() => {
+        anchor = `#${$(this).attr('id')}`;
+      });
+
+      // Add a new error message
+      messageElement = $('<li><a></a></li>');
+      messageElement
+        .attr('id', elementID)
+        .find('a')
+          .attr('href', location.pathname + location.search + anchor)
+          .html(message);
+
+      this.$el.find('ul').append(messageElement);
+
+      // Link back to original input via aria
+      // Respect existing non-error aria-describedby
+      if (!describedBy) {
+        describedBy = elementID;
+      } else if (!describedBy.match(new RegExp(`\\b${elementID}\\b`))) {
+        // Add to end of list if not already present
+        describedBy += ` ${elementID}`;
+      }
+
+      $input.attr('aria-describedby', describedBy);
+    }
+  };
+
+  /**
+   * @func FormStep
+   * @constructor
+   * @param {object} element
+   * @return {object} - The FormStep instance.
+   * @desc Creates a form step.
+   */
+  function FormStep(element) {
+    const self = this;
+
+    this.$el = element instanceof $ ? element : $(element);
+
+    // Find button for this step
+    this.$elButton = $(`.step-button-wrapper[data-for='${this.$el.prop('id')}]`);
+
+    // Has the step been viewed by the user?
+    this.viewed = false;
+
+    // Is the form step valid?
+    // This value is used on form submission, which fails, if any of the steps are invalid.
+    this.valid = false;
+
+    // The internal id of the step. Used for getting the step from the UserForm.steps array.
+    this.id = null;
+
+    this.hide();
+
+    if (CONSTANTS.DISPLAY_ERROR_MESSAGES_AT_TOP) {
+      this.errorContainer = new ErrorContainer(this.$el.find('.error-container'));
+
+      // Listen for errors on the UserForm.
+      userform.$el.on('userform.form.error', (e, validator) => {
+        // The step only cares about errors if it's currently visible.
+        if (!self.$el.is(':visible')) {
+          return;
+        }
+
+        // Add or update each error in the list.
+        $.each(validator.errorList, (i, error) => {
+          self.errorContainer.updateErrorMessage($(error.element), error.message);
+        });
+      });
+
+      // Listen for fields becoming valid
+      userform.$el.on('userform.form.valid', (e, fieldId) => {
+        self.errorContainer.removeErrorMessage(fieldId);
+      });
+    }
+
+    // Ensure that page visibilty updates the step navigation
+    this
+      .$elButton
+      .on('userform.field.hide userform.field.show', () => {
+        userform.$el.trigger('userform.form.conditionalstep');
+      });
+
+    return this;
+  }
+
+  /**
+   * Determine if this step is conditionally disabled
+   *
+   * @returns {Boolean}
+   */
+  FormStep.prototype.conditionallyHidden = () => {
+    // Because the element itself could be visible but 0 height, so check visibility of button
+    return ! this
+      .$elButton
+      .find('button')
+      .is(':visible');
+  };
```

Comment:
This too can be changed to the same format above:
```
// Because the element itself could be visible but 0 height, so check visibility of button
FormStep.prototype.conditionallyHidden = () => (
  !this.$elButton.find('button').is(':visible')
);
```